cvat/apps/documentation/user_guide.md

```diff
@@ -609,38 +610,25 @@ Tracks are created in ``Track mode``
 - _Automatic_ which is created automatically without a person in the loop
 
 ---
-### Workspace — Context menu
+### Workspace
 
-Press the ``Right mouse button`` to see the context menu.
+This is the main field in which drawing and editing objects takes place. 
+In addition the workspace also has the following functions: 
+-   Right-clicking on an object calls up the ``Object card`` - this is an element containing 
+    the necessary controls for changing the label and attributes of the object, as well as the action menu.
 
-Next options are available clicking inside bounding box:
-- ``Copy Object URL`` — copying to the buffer address of an object on the frame
-in the task
-- ``Change Color`` — change color of active shape
-- ``Remove Shape`` — deleting the shape
-- ``Switch Occluded`` — attribute is used if an object is occluded by another
-object or isn't fully visible on the frame. Use the ``Q`` shortcut to set the
-property quickly.
-- ``Switch Lock`` — block editing the active shape
-- ``Enable Dragging`` — (only for polygons) allows to adjust polygons position
-- ``Split`` — allows to split an interpolated track into two separate tracks. This function is the opposite
-of the merge function.
+    ![](static/documentation/images/image138_mapillary_vistas.jpg)
 
-![](static/documentation/images/image089.jpg)
```

Comment:
I see that a couple of images were deleted in documentation. Are they using somewhere or we need to delete these files?